esm-loader.mjs

```diff
@@ -16,11 +17,19 @@ const isFromEsmLoader = (context) =>
   context && context.parentURL && context.parentURL.includes('newrelic/esm-loader.mjs')
 
 const logger = loggingModule.child({ component: 'esm-loader' })
+const esmShimPath = new URL('./lib/esm-shim.mjs', import.meta.url)
+const customEntryPoint = newrelic.agent.config.api.esm.custom_instrumentation_entrypoint
+
+// Hook point within agent for customers to register their custom instrumentation.
+if (customEntryPoint) {
```

Comment:
a lot of other stuff in here but this is the crux of the change.  if a customer sets some config value try to import that file